bin/git-delete-merged-branches

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
-branches=$(git branch --no-color --merged | grep -v "\*" | grep -v master | grep -v svn)
+branches=$(git branch --no-color --merged | grep -v "\*" | grep -vE "\b(master|dev|develop)\b" | grep -v svn)
 if [ -n "$branches" ]
 then
-    echo "$branches" | xargs -n 1 git branch -d
+    echo "$branches" | xargs -rn 1 git branch -d
```

Comment:
prevent error `fatal: branch name required` when there are no merged branches found

Reply:
@KOGI 
It is strange, we have already checked `if [ -n "$branches" ]` before. The `$branches` should not be empty there.

Reply:
Ah, you're right. I must've had an old installation that I copied my changes from.